Formula/acme.rb

```diff
@@ -1,10 +1,8 @@
 class Acme < Formula
   desc "Crossassembler for multiple environments"
-  homepage "https://web.archive.org/web/20150520143433/https://www.esw-heim.tu-clausthal.de/~marco/smorbrod/acme/"
-  url "https://www.mirrorservice.org/sites/ftp.cs.vu.nl/pub/minix/distfiles/backup/acme091src.tar.gz"
-  mirror "http://ftp.lip6.fr/pub/minix/distfiles/backup/acme091src.tar.gz"
-  version "0.91"
-  sha256 "31ed7f9be5cd27100b13d6c3e2faec35d15285542cbe168ec5e1b5236125decb"
+  homepage "https://sourceforge.net/projects/acme-crossass/"
+  url "https://svn.code.sf.net/p/acme-crossass/code-0/trunk", :revision => "97"
+  version "0.96.4"
```

Comment:
Do you have a reference that says 97 is 0.96.4?

Reply:
From the svn log:

```txt
------------------------------------------------------------------------
r97 | marcobaye | 2017-12-22 23:55:36 +0100 (Fri, 22 Dec 2017) | 2 lines

ACME release 0.96.4: Fixed bug in zero page wrap-around warnings. Added "!xor" pseudo op. Added "-I" CLI switch.
```